log/log.go

```diff
@@ -362,3 +367,23 @@ func defaultRobotProperties(msgType, op, key string) LogEntry {
 	}
 	return props
 }
+
+func (l *Log) maskSecretHeaders(headers map[string]string) map[string]string {
+	secretHeaders := map[string]bool{
+		"Authorization": true,
+		"access_token":  true,
+		"itau-chave":    true,
+	}
+
+	maskedHeaders := make(map[string]string)
+
+	for key, value := range headers {
```

Comment:
Opa, talvez fique mais simples a leitura com apenas um IF+continue, e o outro tratamento default 😀 
```suggestion
	for key, value := range headers {
		if secretHeaders[key] {
			maskedHeaders[key] = "*****"
			continue
		}

		maskedHeaders[key] = value
	}
```

Reply:
Eu particularmente prefiro a leitura com o if/else que é o equivalente à um if ternário que provavelmente utilizaríamos em outras linguagens com esse recurso 😬

Reply:
Aopa @brunoglm , eu também prefiro a abordagem com o if/else pq acho que fica mais claro que se não for um caso, é o outro